src/System.Data.SqlClient/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -0,0 +1,1150 @@
+// Copyright (c) Microsoft. All rights reserved.
```

Comment:
We didn't want to expose the AdapterUtil.cs utility methods. 
That's the reason the class has been kept internal and there are two copies of it.